telegram/_telegramobject.py

```diff
@@ -169,28 +170,28 @@ def _get_attrs(
         """
         data = {}
 
-        if not recursive:
-            try:
-                # __dict__ has attrs from superclasses, so no need to put in the for loop below
-                data.update(self.__dict__)
-            except AttributeError:
-                pass
+        # __dict__ has attrs from superclasses, so no need to recompute/duplicate
+        if hasattr(self, "__dict__"):
+            data.update(self.__dict__)  # important when class has no __slots__.
+
+        org_data = tuple(data)  # make a copy of the keys to avoid changing length during iteration
```

Comment:
this is only non-empty if `self.__dict__` is present, right? why not just use `self.__dict__` again instead of creating this tuple?